web/app/js/components/MetricsTable.jsx

```diff
@@ -117,14 +119,16 @@ const columnDefinitions = (resource, namespaces, onFilterClick, showNamespaceCol
       dataIndex: "successRate",
       key: "successRateRollup",
       className: "numeric",
+      width: "120px",
```

Comment:
This width needs to be slightly wider for when the sortable header is present:

![image](https://user-images.githubusercontent.com/9226/45448696-e09c9b00-b687-11e8-8a47-5a83137cd8ec.png)

Testing locally it looks like 130px would work.

Reply:
(note that this applies to the definition in `TopEventTable.jsx` and not the one in `MetricsTable`